Remove PhpMyAdmin/dba from DDEV, fixes #3534 (#4990)
ddev/ddev: docs/content/users/usage/database-management.md

You can also:

* Use [`ddev mysql`](../usage/commands.md#mysql) or `ddev psql` or the `mysql` and `psql` commands inside the `web` and `db` containers.

> * Use many other database browsers to import.

Reviewer: Help someone with a description that’s more specific and actionable:

```suggestion
* Use a [database client](#database-clients) or [database GUI](#database-guis) to import and browse data.
```

If you’d like to use a GUI database client, you’ll need the right connection details and there may even be a command to launch it for you:

> * PhpMyAdmin, formerly built into DDEV core, can be installed with `ddev get ddev/ddev-phpmyadmin`.

Reviewer: ```suggestion
* phpMyAdmin, formerly built into DDEV core, can be installed by running `ddev get ddev/ddev-phpmyadmin`.
```

* Adminer can be installed with `ddev get ddev/ddev-adminer`

> * The [`ddev describe`](../usage/commands.md#describe) command displays the  `Host:` details you’ll need to connect to the `db` container externally, for example if you're using an on-host database browser like SequelAce.

Reviewer 2: ```suggestion
* The [`ddev describe`](../usage/commands.md#describe) command displays the `Host:` details you’ll need to connect to the `db` container externally, for example if you're using an on-host database browser like SequelAce.
```
